Library/Formula/multimarkdown.rb

```diff
@@ -2,9 +2,9 @@ class Multimarkdown < Formula
   desc "Turn marked-up plain text into well-formatted documents"
   homepage "http://fletcherpenney.net/multimarkdown/"
   # Use git tag instead of the tarball to get submodules
-  url "https://github.com/fletcher/MultiMarkdown-4.git", :tag => "4.7.1",
-                                                         :revision => "3083076038cdaceb666581636ef9e1fc68472ff0"
-  head "https://github.com/fletcher/MultiMarkdown-4.git"
+  url "https://github.com/fletcher/MultiMarkdown-5.git", :tag => "v5.0",
```

Comment:
Mind kicking the `tag` and `revision` lines onto their own line? Having them this far over bothers me in terms of quick readability.